packages/web3-common/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './types';
+export * from './web3_base_provider';
```

Comment:
The repo already has spacing denoted using `-` (kebab casing), lets not mix in snake casing as well

Reply:
I am not sure of using `kebab` case for the file names. Due to common JS community convention of package names we had it for the package directory names. 

In `loadstar` its `kebab` case for the package directory names and then `camelCase` for the file names. I think we should keep `snake` case for the files name that's much better in readability and support across platforms.